pkg/ops/netboot.go

```diff
@@ -29,7 +29,7 @@ func StartPixiecore(cloudConfigFile, squashFSfile, address, netbootPort, initrdF
 
 		configFile := cloudConfigFile
 
-		cmdLine := `rd.live.overlay.overlayfs rd.neednet=1 ip=dhcp rd.cos.disable root=live:{{ ID "%s" }} netboot nodepair.enable config_url={{ ID "%s" }} console=tty1 console=ttyS0 console=tty0`
+		cmdLine := `rd.live.overlay.overlayfs selinux=0 rd.neednet=1 ip=dhcp rd.cos.disable root=live:{{ ID "%s" }} netboot nodepair.enable config_url={{ ID "%s" }} console=tty1 console=ttyS0 console=tty0`
```

Comment:
Can you test the Ubuntu image with this change? I remember there being issues trying to start it without selinux enabled

Reply:
any specific version? latest?

Reply:
yeah latest as long as it's the `ubuntu` only flavor, not lts

Reply:
will test, thanks for the tip!

Reply:
@mauromorales used latest core-ubuntu:v2.3.1 artifact and was able to boot and install with no issues!

Reply:
fantastic!